test/ns-test.js

```diff
@@ -370,6 +370,12 @@ describe('RootServer DNSSEC', function () {
                         wire.types.RRSIG,
                         wire.types.NSEC]],
       [TYPE_MAP_TXT,   [wire.types.TXT,
+                        wire.types.RRSIG,
+                        wire.types.NSEC]],
+      [TYPE_MAP_A,     [wire.types.A,
+                        wire.types.RRSIG,
+                        wire.types.NSEC]],
+      [TYPE_MAP_AAAA,  [wire.types.AAAA,
```

Comment:
I just realized this little test doesn't check for extra unexpected types in the maps, but eh I think that's ok

Reply:
This test checks if the two bitmaps are identical right? I added type A/AAAA to those checks. If there are extra unexpected types it should fail?